unitTests/ui/StandaloneSearchInterfaceTest.ts

```diff
@@ -1,33 +1,165 @@
 import * as Mock from '../MockEnvironment';
-import { StandaloneSearchInterface } from '../../src/ui/SearchInterface/SearchInterface';
+import { StandaloneSearchInterface, IStandaloneSearchInterfaceOptions } from '../../src/ui/SearchInterface/SearchInterface';
+import { QueryEvents, $$, StandaloneSearchInterfaceEvents } from '../../src/Core';
+import { INewQueryEventArgs } from '../../src/events/QueryEvents';
+import { ExecutionPlan } from '../../src/rest/Plan';
+import { FakeResults } from '../Fake';
 import { analyticsActionCauseList } from '../../src/ui/Analytics/AnalyticsActionListMeta';
+
 export function StandaloneSearchInterfaceTest() {
   describe('StandaloneSearchInterface', () => {
     let cmp: StandaloneSearchInterface;
+    let options: IStandaloneSearchInterfaceOptions;
     let windoh = window;
 
     beforeEach(() => {
       windoh = Mock.mockWindow();
+      options = {
+        searchPageUri: 'https://connect.coveo.com/s/global-search/'
+      };
+      initializeComponent();
+    });
+
+    function initializeComponent() {
+      cmp = new StandaloneSearchInterface(document.createElement('div'), options, undefined, windoh);
+    }
+
+    it(`on a newQuery event
+    should call handleRedirect`, () => {
+      spyOn(cmp, 'handleRedirect');
+      $$(cmp.root).trigger(QueryEvents.newQuery);
+      expect(cmp.handleRedirect).toHaveBeenCalled();
```

Comment:
I prefer to use a stricter check on the number of times the function was called. This asserts we are not calling the function multiple times. e.g.

```
expect(cmp.handleRedirect).toHaveBeenCalledTimes(1);
```